vispy/app/backends/_qt.py

```diff
@@ -590,7 +590,6 @@ def _vispy_set_current(self):
         if self._vispy_canvas is None:
             return  # todo: can we get rid of this now?
         if self.isValid():
-            self._vispy_canvas.set_current()  # Mark as current
```

Comment:
This seems weird to me -- inside the backend's `_vispy_set_current`, this seems to make it so that it no longer makes the Canvas's GLIR queue current? (Right?)